EE_Mini_Cart_Table_Line_Item_Display_Strategy.php

```diff
@@ -302,8 +306,7 @@ private function _sub_total_row( EE_Line_Item $line_item, $text = '' ) {
 				// total td
 			$html .= EEH_HTML::td( $text, '', 'total_currency total jst-rght', '', ' colspan="3"' );
 			// total td
-			$total = $line_item->total();
-			$html .= EEH_HTML::td( EEH_Template::format_currency( $total, false, false ), '', 'total jst-rght' );
+			$html .= EEH_HTML::td($line_item->total_no_code(), '', 'total jst-rght' );
```

Comment:
```suggestion
			$html .= EEH_HTML::td($line_item->total_no_code(), '', 'total jst-rght');
```
FYI, Not important really.

Reply:
good eye, thnx, will fix